src/nativescript/http.ts

```diff
@@ -74,11 +74,16 @@ export class HttpMiddleware extends Middleware {
     };
     return (HttpRequest(options) as any)
       .then((response) => {
+        const contentType = response.headers['Content-Type'] || response.headers['content-type'];
         let data = response.content.raw;
 
-        try {
-          data = response.content.toString();
-        } catch (e) {}
+        if (contentType) {
+          if (contentType.indexOf('application/json') === 0 || contentType.indexOf('application/xml') === 0) {
+            try {
+              data = response.content.toString();
+            } catch (e) {}
```

Comment:
Does it make sense here to have a log message, so that we know for what situations we may be failing to call `toString()` when the content type is either `application/json` or `application/xml`?

Reply:
I removed the check of the `Content-Type` header.